src/git/commit.js

```diff
@@ -18,7 +18,7 @@ function commit(sh, repoPath, message, options, done) {
   // On windows we must use an array in gulp-git instead of a string because
   // command line parsing works differently
   if(os.platform()=="win32") {
-    operatingSystemNormalizedMessage = escapedMessage.split(/\r?\n/);
+    operatingSystemNormalizedMessage = escapedMessage.replace(/\r/g, '');
```

Comment:
gulp-git 1.9.0 added support for multi-line messages. I guess I forgot to change that comment